Core/src/org/sleuthkit/autopsy/contentviewers/ContextViewer.java

```diff
@@ -0,0 +1,427 @@
+                .addContainerGap()
+                .addComponent(jLabel1)
+                .addPreferredGap(javax.swing.LayoutStyle.ComponentPlacement.RELATED)
+                .addGroup(layout.createParallelGroup(javax.swing.GroupLayout.Alignment.BASELINE)
+                    .addComponent(jSourceNameLabel)
+                    .addComponent(jSourceTextLabel))
+                .addGap(18, 18, 18)
+                .addComponent(jSourceGoToResultButton)
+                .addGap(0, 203, Short.MAX_VALUE))
+        );
+    }// </editor-fold>//GEN-END:initComponents
+
+    private void jSourceGoToResultButtonActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_jSourceGoToResultButtonActionPerformed
+
+        final DirectoryTreeTopComponent dtc = DirectoryTreeTopComponent.findInstance();
+
+        // Navigate to the source context artifact.
+        if (sourceContextArtifact != null) {
+            dtc.viewArtifact(sourceContextArtifact);
+        }
+
+    }//GEN-LAST:event_jSourceGoToResultButtonActionPerformed
+
+    @Override
+    public void setNode(Node selectedNode) {
+        if ((selectedNode == null) || (!isSupported(selectedNode))) {
+            resetComponent();
+            return;
+        }
+
+        AbstractFile file = selectedNode.getLookup().lookup(AbstractFile.class);
+        try {
+            populateSourceContextData(file);
+        } catch (NoCurrentCaseException | TskCoreException ex) {
+            logger.log(Level.SEVERE, "Exception displaying context for file {0}", file); //NON-NLS
+        }
+    }
+
+    @Override
+    public String getTitle() {
+        return Bundle.ContextViewer_title();
+    }
+
+    @Override
+    public String getToolTip() {
+        return Bundle.ContextViewer_toolTip();
+    }
+
+    @Override
+    public DataContentViewer createInstance() {
+        return new ContextViewer();
+    }
+
+    @Override
+    public Component getComponent() {
+        return this;
+    }
+
+    @Override
+    public void resetComponent() {
+        setSourceName("");
+        setSourceText("");
+    }
+
+    @Override
+    public boolean isSupported(Node node) {
+
+        // check if the node has an abstract file and the file has any context defining artifacts.
+        if (node.getLookup().lookup(AbstractFile.class) != null) {
+            AbstractFile abstractFile = node.getLookup().lookup(AbstractFile.class);
+            for (BlackboardArtifact.ARTIFACT_TYPE artifactType : SOURCE_CONTEXT_ARTIFACTS) {
+                List<BlackboardArtifact> artifactsList;
+                try {
+                    artifactsList = abstractFile.getArtifacts(artifactType);
+                    if (!artifactsList.isEmpty()) {
+                        return true;
+                    }
+                } catch (TskCoreException ex) {
+                    logger.log(Level.SEVERE, "Exception while looking up context artifacts for file {0}", abstractFile); //NON-NLS
```

Comment:
Pass the exception to the log message so that the stack trace is output to the log.